kubernetes-maven-plugin/it/src/it/simple-with-route-flag-false/invoker.properties

```diff
@@ -13,5 +13,5 @@
 #
 
 invoker.goals.1=clean k8s:resource
-invoker.mavenOpts=-Djkube.verbose -Djkube.mode=openshift -Dfabric8.openshift.generateRoute=false
+invoker.mavenOpts=-Djkube.verbose -Dfabric8.openshift.generateRoute=false
```

Comment:
Probably the second option `-Dfabric8.openshift.generateRoute=false` has no effect (if it does then something is wrong), we should remove it.